src/content/app/tools/blast/views/blast-submission-results/components/job-parameters/JobParameters.scss

```diff
@@ -0,0 +1,58 @@
+@import 'src/styles/common';
+
+.submissionParameters {
+  display: grid;
+  grid-template-areas:
+    'main-top aside-top'
+    'main-bottom aside-bottom';
+  grid-template-columns: min-content 1fr;
+  grid-template-rows: 20px auto;
+  align-items: start;
+  column-gap: 70px;
+
+  .copySequence {
+    justify-self: flex-end;
+    grid-area: main-top;
+  }
+
+  .sequence {
+    width: calc(60ch + 14px);
+    grid-area: main-bottom;
+    margin-top: 10px;
+    min-height: 170px;
+    max-height: 430px;
+    font-family: $font-family-monospace;
+    color: $dark-grey;
+    overflow-y: auto;
+    overflow-x: hidden;
```

Comment:
Why do we need this? It could be useful for a long header, I suppose; but `sequenceValue` that you are displaying in this container won't have the header.

Reply:
I couldn't replicate the horizontal scrollbar anymore so I've removed `overflow-x`.